ee/clickhouse/views/actions.py

```diff
@@ -133,13 +133,11 @@ def _process_content_sql(team: Team, entity: Entity, filter: Filter):
         cohort = Cohort.objects.get(pk=filter.breakdown_value)
         person_filter, person_filter_params = format_filter_query(cohort)
         person_filter = "AND distinct_id IN ({})".format(person_filter)
-    elif (
-        filter.breakdown_type == "person"
-        and isinstance(filter.breakdown, str)
-        and isinstance(filter.breakdown_value, str)
-    ):
-        person_prop = Property(**{"key": filter.breakdown, "value": filter.breakdown_value, "type": "person"})
-        filter.properties.append(person_prop)
+    elif filter.breakdown_type and isinstance(filter.breakdown, str) and isinstance(filter.breakdown_value, str):
```

Comment:
this statement was testing for person breakdowns specifically before but now it will make sure to add either person or event breakdown values to the filtering